packages/components/src/step/step.tsx

```diff
@@ -0,0 +1,74 @@
+import { Stack, styled } from '@tamagui/core'
+
+import { Text } from '../text'
+
+import type { ColorTokens } from '@tamagui/core'
+
+// todo?: rename netural to default
+export type StepVariants = 'neutral' | 'complete' | 'active'
```

Comment:
Left a comment in https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n?node-id=18126:5270&mode=design#531010703.